clang/test/CodeGen/integer-wrap.c

```diff
@@ -0,0 +1,77 @@
+// Check that -fsanitize=signed-integer-wrap instruments with -fwrapv
+// RUN: %clang_cc1 -fwrapv -triple x86_64-apple-darwin -emit-llvm -o - %s -fsanitize=signed-integer-wrap | FileCheck %s --check-prefix=CHECKSIW
+
+// Check that -fsanitize=signed-integer-overflow doesn't instrument with -fwrapv
+// RUN: %clang_cc1 -fwrapv -triple x86_64-apple-darwin -emit-llvm -o - %s -fsanitize=signed-integer-overflow | FileCheck %s --check-prefix=CHECKSIO
+
+extern volatile int a, b, c;
+
+// CHECKSIW-LABEL: define void @test_add_overflow
```

Comment:
I believe @tschuett was saying that to be confident we are instrumenting the arithmetic (and the correct operands therein) we should check the overflow bit usage in the branch instruction. The only way I could think of doing this was capturing the call to the overflow intrinsic which returns the value and the overflow bit. Is there a way to minimize my CHECKs?

Reply:
I was thinking to be certain that no instrumentation exists we just need to check that no call to the handler exists. The idea being that it's easier to disprove something than to prove it.

Reply:
Resolved in [5497e8b](https://github.com/llvm/llvm-project/pull/80089/commits/5497e8bc6849bf64c1158ff16b4aa04fd9141920) since the tests have been moved there is no issue with LABEL statements or prefixes.